src/compose/ComposeBox.js

```diff
@@ -201,7 +208,8 @@ class ComposeBox extends PureComponent<Props, State> {
       isMessageFocused: false,
       isMenuExpanded: false,
     });
-    setTimeout(this.updateIsFocused, 200); // give a chance to the topic input to get the focus
+    // give a chance to the topic input to get the focus
+    this.updateIsFocusedTimeout = setTimeout(this.updateIsFocused, 200);
```

Comment:
This will clobber an existing value of `this.updateIsFocusedTimeout`; if two blur events are handled in quick succession before the unmount, the first one's invocation will still be called.

Reply:
Thanks for catching this!